clr-k8s-examples/create_stack.sh

```diff
@@ -50,53 +58,150 @@ function cluster_init() {
 }
 
 function kata() {
-	# Install kata artifacts using kata-deploy
-	kubectl apply -f 8-kata/deploy/kata-rbac.yaml
-	kubectl apply -f 8-kata/deploy/kata-deploy.yaml
-	kubectl apply -f 8-kata/
+	KATA_VER=${1:-1.8.0-kernel-config}
+	KATA_URL="https://github.com/kata-containers/packaging.git"
+	KATA_DIR="8-kata"
+	get_repo "${KATA_URL}" "${KATA_DIR}/overlays/${KATA_VER}"
```

Comment:
Can we apply the kata files without actually cloning the repository? Say, `kubectl apply -f https://raw.githubusercontent.com/kata-containers/packaging/1.8.0-kernel-config/kata-deploy/kata-deploy.yaml` or an equivalent of that?

Reply:
We can but it breaks the pattern we are using w kustomize and would have to change back if we need to patch anything. Not sure its worth it, is the advantage just that its quicker?

Reply:
plus you have to maintain a list of files for `kubectl apply` per version and that is already taken care of in new approach w Kustomize.

Reply:
yeah, was just going for quicker, not adding stuff in a folder etc.. but i guess the alternate is not exactly super future proof either. I am ok with it being what you have for now.